reporter/src/funTest/kotlin/reporters/CycloneDxReporterFunTest.kt

```diff
@@ -36,13 +41,25 @@ import org.ossreviewtoolkit.utils.ORT_NAME
 
 class CycloneDxReporterFunTest : WordSpec({
     val options = mapOf("single.bom" to "true")
+    val outputDir = createTempDirectory("$ORT_NAME-${javaClass.simpleName}").toFile().apply { deleteOnExit() }
 
     "A generated BOM" should {
-        "be valid according to schema version 1.2" {
-            val outputDir = createTempDirectory("$ORT_NAME-${javaClass.simpleName}").toFile().apply { deleteOnExit() }
-            val bomFile = CycloneDxReporter().generateReport(ReporterInput(ORT_RESULT), outputDir, options).single()
+        "be valid XML according to schema version 1.2" {
+            val xmlOptions = options + mapOf("output.file.formats" to "xml")
+            val bomFile = CycloneDxReporter().generateReport(ReporterInput(ORT_RESULT), outputDir, xmlOptions).single()
 
+            bomFile shouldBe aFile()
+            bomFile shouldNotBe emptyFile()
```

Comment:
The assertions in line 51 and 52 should not be needed. Consider removing them.

..same below.

Reply:
They are not strictly needed, but they give much better clues what went wrong, as in case of an empty file the error message from the validator is rather non-telling.